packages/lab/src/multiline-input/MultilineInput.css

```diff
@@ -0,0 +1,237 @@
+  --multilineInput-activationIndicator-borderWidth: var(--salt-size-border);
+
+  background: var(--multilineInput-background-disabled);
+  cursor: var(--salt-editable-cursor-disabled);
+  color: var(--salt-text-primary-foreground-disabled);
+}
+
+/* Style for activation indicator */
+.saltMultilineInput-activationIndicator {
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  position: absolute;
+  border-bottom: var(--multilineInput-activationIndicator-borderWidth) var(--multilineInput-borderStyle) var(--multilineInput-borderColor);
+}
+
+/* Style applied to adornment containers */
+.saltMultilineInput-endAdornmentContainer,
+.saltMultilineInput-startAdornmentContainer {
+  align-items: center;
+  display: inline-flex;
+  min-height: calc(var(--salt-size-base) + var(--salt-spacing-100));
+  column-gap: var(--salt-spacing-100);
+}
+
+/* Style applied to start adornment container */
+.saltMultilineInput-startAdornmentContainer {
+  align-self: self-start;
+  padding-right: var(--salt-spacing-100);
+}
+
+/* Style applied to end adornment container */
+.saltMultilineInput-endAdornmentContainer {
+  align-self: self-end;
+  padding-left: var(--salt-spacing-100);
+}
+
+/* Style applied if `fullBorder={true}` */
+.saltMultilineInput-fullBorder {
+  --multilineInput-border: var(--salt-size-border) var(--salt-container-borderStyle) var(--multilineInput-borderColor);
+  --multilineInput-paddingLeft: var(--salt-spacing-100);
+  --multilineInput-activationIndicator-borderWidth: 0;
+}
+
+/* Style applied if active or focused when `fullBorder={true}` */
+.saltMultilineInput-fullBorder:active,
+.saltMultilineInput-fullBorder.saltMultilineInput-focused {
+  --multilineInput-activationIndicator-borderWidth: var(--salt-editable-borderWidth-active);
+}
+
+/* Styling when focused if `disabled={true}` or `readOnly={true}` when `fullBorder={true}` */
+.saltMultilineInput-fullBorder.saltMultilineInput-readOnly,
+.saltMultilineInput-fullBorder.saltMultilineInput-focused.saltMultilineInput-readOnly,
+.saltMultilineInput-fullBorder.saltMultilineInput-readOnly:hover,
+.saltMultilineInput-fullBorder.saltMultilineInput-focused.saltMultilineInput-disabled,
+.saltMultilineInput-fullBorder.saltMultilineInput-disabled:hover {
+  --multilineInput-paddingLeft: var(--salt-spacing-100);
+  --multilineInput-activationIndicator-borderWidth: 0;
+}
+
+/* Style applied to status adornment container */
+.saltMultilineInput-statusAdornmentContainer {
+  align-self: self-end;
+  display: inline-flex;
+  min-height: calc(var(--salt-size-base) + var(--salt-spacing-100));
+}
+
+/* Style applied to button start adornment if first child */
+.saltMultilineInput-startAdornmentContainer .saltButton:first-child {
+  margin-left: calc(var(--salt-spacing-50) * -1);
+}
+
+/* Style applied to button start adornment if last child */
+.saltMultilineInput-endAdornmentContainer .saltButton:last-child {
+  margin-right: calc(var(--salt-spacing-50) * -1);
+}
```

Comment:
sorry for being a pain, but would you mind moving the adornments styles closer together :P.

Reply:
done